src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java

```diff
@@ -76,7 +77,7 @@ protected void onEvent(GHEvent event, String payload) {
         return new Function<GHEventsSubscriber, Set<GHEvent>>() {
             @Override
             public Set<GHEvent> apply(GHEventsSubscriber subscriber) {
-                return defaultIfNull(subscriber.events(), Collections.<GHEvent>emptySet());
+                return defaultIfNull(notNull(subscriber).events(), Collections.<GHEvent>emptySet());
```

Comment:
what am I missing here?
[`notNull(Object)`](https://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/Validate.html#notNull(T)) will throw a `NullPointerException` if the Object is `null` - so no bug has been fixed here - just hidden under the carpet.